dynamiq/nodes/agents/base.py

```diff
@@ -493,14 +512,57 @@ def _get_tool(self, action: str) -> Node:
             )
         return tool
 
-    def _run_tool(self, tool: Node, tool_input: str, config, **kwargs) -> Any:
+    def _apply_parameters(self, merged_input: dict, params: dict, source: str, debug_info: list = []):
```

Comment:
debug_info: list = None is correct

But in code please make:
`debug_info  = [] if debug_info is None or debug_info`